### PR TITLE
Standalone to consumer changeover - recycle existing transfer channel

### PR DESCRIPTION
**Pull request by smarshall-spitzbart**
# Description

Allows previously standalone chains to use the already existing transfer channel from the standalone chain

## Linked issues

Closes: https://github.com/cosmos/interchain-security/issues/782

Epic: #756 

## Type of change

If you've checked more than one of the first three boxes, consider splitting this PR into multiple PRs!

- [x] `Feature`: Changes and/or adds code behavior, irrelevant to bug fixes
- [ ] `Fix`: Changes and/or adds code behavior, specifically to fix a bug
- [ ] `Refactor`: Changes existing code style, naming, structure, etc.
- [ ] `Testing`: Adds testing
- [ ] `Docs`: Adds documentation

## Regression tests

Existing handshake tests like `TestOnChanOpenAck`

## New behavior tests

CRUD unit tests, and new integration test in `changeover.go`

## Versioning Implications

- [x] This PR will affect [semantic versioning as defined for ICS](../CONTRIBUTING.md#semantic-versioning)

If the above box is checked, which version should be bumped?

- [ ] `MAJOR`: Consensus breaking changes to both the provider and consumers(s), including updates/breaking changes to IBC communication between provider and consumer(s)
- [ ] `MINOR`: Consensus breaking changes which affect either only the provider or only the consumer(s)
- [x] `PATCH`: Non consensus breaking changes

## Targeting

Please select one of the following:

- [x] This PR is only relevant to main
- [ ] This PR is relevant to main, and should also be back-ported to ____ (ex: v1.0.0 and v1.1.0)
- [ ] This PR is only relevant to ____ (ex: v1.0.0, v1.1.0, and v1.2.0)

### CODE REVIEW

**Review by MSalopek — APPROVED** [review]
Looks good!

e2e tests will be added at a later point

File: x/ccv/consumer/keeper/validators.go [hunk 1]
```diff
@@ -111,7 +111,7 @@ func (k Keeper) Slash(ctx sdk.Context, addr sdk.ConsAddress, infractionHeight, p
 
 	// If this is a previously standalone chain and infraction happened before the changeover was completed,
 	// slash only on the standalone staking keeper.
-	if k.IsPrevStandaloneChain() && infractionHeight < k.FirstConsumerHeight(ctx) {
+	if k.IsPrevStandaloneChain(ctx) && infractionHeight < k.FirstConsumerHeight(ctx) {
 		k.standaloneStakingKeeper.Slash(ctx, addr, infractionHeight, power, slashFactor, stakingtypes.InfractionEmpty)
```
Comment (jtremback) [inline x/ccv/consumer/keeper/validators.go:115]: We need to dig into how exactly consumer chains using the democracy module or other things will populate standaloneStakingKeeper

**Review by jtremback — APPROVED** [review]
LGTM

File: x/ccv/consumer/keeper/validators.go (hunk 1 above)
Comment (mpoke) [reply x/ccv/consumer/keeper/validators.go:115]: What do you mean? That's the staking module with the native token from when the chain was a standalone chain.

**Pull request merged**